scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json

```diff
@@ -97,6 +109,12 @@
         "dcrName":"[if(greater(length(variables('dcrNameFull')), 64), substring(variables('dcrNameFull'), 0, 64), variables('dcrNameFull'))]",
         "associationName":  "ContainerInsightsExtension",
         "dataCollectionRuleId": "[resourceId(variables('clusterSubscriptionId'), variables('clusterResourceGroup'), 'Microsoft.Insights/dataCollectionRules', variables('dcrName'))]",
+        "dceName": "[variables('dcrName')]",
+        "dceAssociationName": "configurationAccessEndpoint",
```

Comment:
Is it okay if the dceAssociationName is same always?  Is there a scenario where the same cluster can have multiple dce associations?

Reply:
> Is it okay if the dceAssociationName is same always? Is there a scenario where the same cluster can have multiple dce associations?

Great question. The current implementation of AMCS, dceAssociatioName has to be constant and it has to be "configurationAccessEndpoint".  Let me check with AMCS to understand the behavior if the cluster has multiple DCE-Associations.

Reply:
As discussed with AMCS team. DCE association for configuration is singleton which is by design.